pardot_theme/version_switch.html

```diff
@@ -0,0 +1,6 @@
+{% set url = current_page.abs_url.split('/') %}
+{% if 'version-3' in current_page.abs_url %}
+<a href="{{base_url}}/kb/api-version-4/{{url[3]}}">Looking for version 4?</a>
+{% elif 'version-4' in current_page.abs_url %}
+<a href="{{base_url}}/kb/api-version-3/{{url[3]}}">Looking for version 3?</a>
+{% endif %}
```

Comment:
certainly not pretty, but it adds the v3 <-> v4 links to the tops of all v3/v4 pages, based on the folder the file is in.